tools/bumper/cnao_repo_commands_test.go

```diff
@@ -55,9 +55,7 @@ var _ = Describe("Testing internal git CNAO Repo", func() {
 
 		DescribeTable("and checking isPrAlreadyOpened function",
 			func(r isPrAlreadyOpenedParams) {
-				defer func(path string) {
-					Expect(os.RemoveAll(path)).To(Succeed())
-				}(gitCnaoRepo.gitRepo.localDir)
+				DeferCleanup(os.RemoveAll, gitCnaoRepo.gitRepo.localDir)
```

Comment:
I think that now, after moving to DeferCleanup, it's possible, and so it is proffered, to check the error; e.g.
```suggestion
				DeferCleanup(func(){
								Expect(os.RemoveAll(gitCnaoRepo.gitRepo.localDir)).To(Succeed())
				})
```

Reply:
I would say better to keep it simple

Reply:
I would say that in go, you must check errors

Reply:
@nunnatsa but it looks like ginkgo already [checks](https://github.com/onsi/ginkgo/blob/master/internal/node.go#L586) for error: